.vscode/settings.json

```diff
@@ -1,3 +1,3 @@
 {
     "haskell.formattingProvider": "fourmolu"
-}
+}
```

Comment:
Why did you remove the newline here?

Reply:
I have no idea - it was probably done by vscode and I didn't notice.

This is a bit of an issue since I have my own local settings in this file and I have to remove them everytime I check out branches.... oh well.

Reply:
If it's done by VSCode, then don't care about it. Just VSCode being stupid I suppose.

Reply:
What are your own local settings? Would it be best if we remove the entire file entirely again? You might want to discuss this with @srid , I don't use VSCode.